tutorials/relay.md

```diff
@@ -80,16 +80,13 @@ same GCP account, and that way be able to ssh to the relay VM.
 1. Go to [the GCP console](https://console.cloud.google.com/compute/instances).
 1. Click **CREATE INSTANCE**.
 1. Set these parameters (accept the default for all the others):
-
-| Heading | Parameter | Value
-|---------|-----------|-------
-| Machine configuration | Machine type | e2-micro
-| Boot disk | Images | Debian GNU/Linux 10 (buster)
-| Identity and API access | Access scopes | All full access to all Cloud APIs
-
+<table>
+<tr><th>Heading</th><th>Parameter</th><th>Value</th></tr>
```

Comment:
it was formatted as a tble. why did you change that?

Reply:
Because a markdown table resets the step numbering

![image](https://user-images.githubusercontent.com/12722969/103158117-03cbb580-4780-11eb-8fdc-ad38a30f4cf7.png)

Reply:
markdown reset numbering only if the table starts at column 1.
if the table is indented (as it should) - numbering continues.
html tables in document are un-editable stuff..

Reply:
Thank you. I'm fixing it.